rust/src/transport/mqtt/mqtt_client.rs

```diff
@@ -6,28 +6,28 @@
 //
 // Author: Frédéric GARDES <frederic.gardes@orange.com> et al.
 // Software description: This Intelligent Transportation Systems (ITS) [MQTT](https://mqtt.org/) client based on the [JSon](https://www.json.org) [ETSI](https://www.etsi.org/committee/its) specification transcription provides a ready to connect project for the mobility (connected and autonomous vehicles, road side units, vulnerable road users,...).
-use crate::analyse::item::Item;
-use crate::reception::exchange::Exchange;
+
+use crate::transport::mqtt::topic::Topic;
+use crate::transport::packet::Packet;
 use log::{debug, error, info, trace, warn};
 use rumqttc::{AsyncClient, Event, EventLoop, MqttOptions, QoS, SubscribeFilter};
 
-pub(crate) struct Client {
+pub(crate) struct MqttClient {
```

Comment:
This is normal because this struct is just introduced in this branch and will be used when analysis pipeline will be added (check [client branch][1])
So yes, as 2.0.0 is some kind of a working branch I think these warnings can be merged in this branch, but, all warnings must be fixed in the end when merging 2.0.0 in master

Note that this remark applies to warnings like `unused`, other more important warning have to be fixed when merging in 2.0.0

[1]: https://github.com/nbuffon/its-client/tree/rust_client/